data/mods/Aftershock/maps/furniture.json

```diff
@@ -425,5 +425,622 @@
         { "item": "motor_small", "count": 1 }
       ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_mil_comms_terminal",
+    "name": "military SICC terminal",
+    "description": "Console interface for a Standard Interstellar Communications Cluster.  With the machine connected to it broken, best salvaged for parts.",
```

Comment:
```suggestion
    "description": "Console interface for a Standard Interstellar Communications Cluster.  With the machine connected to it broken, it can only be salvaged for parts.",
```